src/Matcher.js

```diff
@@ -0,0 +1,72 @@
+const { getData } = require('./Fetcher');
+const { writeJson, readJson } = require('./Json');
+const { crawlDataSkin } = require('./Crawler');
+const { credentials } = require('../config.json');
+const log = require('node-pretty-log');
+
+
+const getUsedWidgets = slots => {
+    let widgets = [];
+    Object.values(slots).forEach(slot => {
```

Comment:
Here you could just return this function if you use a `.map`, since it's gonna return an array. Then you could get rid of the `widgets` variable